.dockerignore

```diff
@@ -0,0 +1,5 @@
+__pycache__/
```

Comment:
Did we include the compiled Python files before? So if we stop doing this, will this increase the start-up time of the container? I'm not particularly concerned about this, just trying to understand the change.

Reply:
Good catch!

Is startup time critical?

Also, I understand that object files are built on demand, and I don't think we used to execute the python modules before publishing the image. So AFAIU this change wouldn't affect much the content of the resulting image

Reply:
> Also, I understand that object files are built on demand, and I don't think we used to execute the python modules before publishing the image. So AFAIU this change wouldn't affect much the content of the resulting image

Yeah @leplatrem, this sounds right. We don't run any python before we build the images, so `__pycache__` directories won't be present.

Reply:
Start-up time can be relevant for auto-scaling, which we do for kintowe, but not for Remote Settings. Since we are only going to use newer images for Remote Settings, start-up times are not really relevant.

The Python standard library offers the `compileall` module, which can be called from the command-line as `python -m compileall <directory>` to recursively compile all Python files in a directory. This is sometimes used for Docker images, but I dont' think it's necessary here. I just wanted this to be an intentional decision rather than an accidental one.

Reply:
@smarnach this was intentional in that I did include this change to make local builds faster. I think this change does not affect on how we build / publish images in CI though, based on @leplatrem's comment.